sway/desktop/xwayland.c

```diff
@@ -25,6 +25,15 @@ static void unmanaged_handle_request_configure(struct wl_listener *listener,
 		ev->width, ev->height);
 }
 
+static void unmanaged_handle_request_fullscreen(struct wl_listener *listener,
+		void *data) {
+	struct sway_xwayland_view *xwayland_view =
+		wl_container_of(listener, xwayland_view, request_fullscreen);
```

Comment:
Unmanaged surfaces aren't views, so this will fail. Unamanged surfaces are typically popups and menus, so I don't think we want to be able to put them fullscreen.